lib/apartment/adapters/abstract_adapter.rb

```diff
@@ -164,9 +174,25 @@ def load_or_abort(file)
         end
       end
 
+      def rescuable_exceptions
+        [ActiveRecord::ActiveRecordError] + Array(rescue_from)
+      end
+
+      def rescue_from
+        []
+      end
+
       def raise_connect_error!(tenant, exception)
         raise TenantNotFound, "Error while connecting to tenant #{tenant}: #{exception.message}"
       end
+
+      def raise_create_tenant_error!(tenant, exception)
```

Comment:
I'm a bit confused why the original fork dropped this code?

Reply:
🤷‍♂️ Some of the tenant create/drop actions weren't fully fleshed out in that fork. May have been that [this commit](https://github.com/mikecmpbll/apartment/pull/1/commits/6431e58c31207bfa23b11f38711afc84d7a0b8f7) just rewrote AbstractAdapter completely and didn't get around to bringing anything similar back in.